cibuild.cmd

```diff
@@ -42,8 +42,8 @@ if not "%BuildTimeLimit%" == "" (
 ) else (
     set RunProcessWatchdog=false
 )
-    
-call "C:\Program Files (x86)\Microsoft Visual Studio 14.0\Common7\Tools\VsDevCmd.bat" || goto :BuildFailed
+
+call "%RoslynRoot%SetDevCommandPrompt.cmd" || goto :BuildFailed
```

Comment:
I inherited this script from Brett's prior work. @brettfo @jasonmalinowski any reason why we always [force using the Dev15 build tools](https://github.com/dotnet/roslyn/pull/12586/files#diff-b984b1ebf1f5adb398c55e1d2d3b0fd1R3)  if Dev15 is installed, instead of first checking if we are building on a Dev14 command prompt? If this was just an oversight, I can fix it.